### PR TITLE
LociPartitioning cleanups

### PR DESCRIPTION
**Pull request by ryan-williams**
see commits

### CODE REVIEW

File: src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioner.scala [hunk 1]
```diff
@@ -53,24 +60,31 @@ trait LociPartitionerArgs
...
   }
 
   @Args4JOption(
     name = "--partitioning-stats",
-    usage = "Compute additional statistics about the partitioned loci and reads; causes additional Spark jobs to be run, " +
-      "so should be disabled in performance-critical environments. Default: false.",
+    usage =
+      "Compute additional statistics about the partitioned loci and reads; causes additional Spark jobs to be run, " +
+        "so should be disabled in performance-critical environments. Default: false.",
     handler = classOf[BooleanOptionHandler]
```
Comment (arahuja) [inline src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioner.scala]: OOC - does this add something on top of the usual boolean arg situation?

File: src/main/scala/org/hammerlab/guacamole/loci/partitioning/CappedRegionsPartitioner.scala [hunk 2]
```diff
@@ -131,4 +73,69 @@ class CappedRegionsPartitioner[R <: ReferenceRegion: ClassTag](regions: RDD[R],
...
+        val total = runs.map(_._1.toLong).sum
+        s"${runs.length} runs, $total loci (avg %.1f): $rs".format(total.toDouble / runs.length)
+      }
+    }
+
+    val totalLoci = validLoci + invalidLoci
+
+    val loci = lociBroadcast.value
+
+    progress(
```
Comment (arahuja) [inline src/main/scala/org/hammerlab/guacamole/loci/partitioning/CappedRegionsPartitioner.scala]: Any reason to come back to `progress` here vs going more in the direction of #551

File: src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioner.scala (hunk 1 above)
Comment (ryan-williams) [inline src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioner.scala]: ha no I guess that is the default handler anyway, removed, good catch!

File: src/main/scala/org/hammerlab/guacamole/loci/partitioning/CappedRegionsPartitioner.scala (hunk 2 above)
Comment (ryan-williams) [inline src/main/scala/org/hammerlab/guacamole/loci/partitioning/CappedRegionsPartitioner.scala]: well, this `progress` statement was already there, and it's still consistently used around the codebase, don't want to start mixing log-formats, just going to switch it all in one fell swoop at some point

**Pull request merged**